test/e2e/storage/testsuites/provisioning.go

```diff
@@ -690,3 +710,47 @@ func prepareDataSourceForProvisioning(
 
 	return dataSourceRef, cleanupFunc
 }
+
+func preparePVCDataSourceForProvisioning(
+	node framework.NodeSelection,
+	client clientset.Interface,
+	dynamicClient dynamic.Interface,
+	source *v1.PersistentVolumeClaim,
+	class *storagev1.StorageClass,
+) (*v1.TypedLocalObjectReference, func()) {
+	var err error
+	if class != nil {
+		ginkgo.By("[Initialize dataSource]creating a StorageClass " + class.Name)
+		_, err = client.StorageV1().StorageClasses().Create(class)
+		framework.ExpectNoError(err)
+	}
+
+	ginkgo.By("[Initialize dataSource]creating a source PVC")
+	sourcePVC, err := client.CoreV1().PersistentVolumeClaims(source.Namespace).Create(source)
+	framework.ExpectNoError(err)
+
+	ginkgo.By("[Initialize dataSource]checking the source PVC")
+	// Get new copy of the source
+	_, err = client.CoreV1().PersistentVolumeClaims(sourcePVC.Namespace).Get(sourcePVC.Name, metav1.GetOptions{})
```

Comment:
unsure if we need this call? We only need pvc.namespace + name for this test, which should already be set in L729.

Reply:
removed